### ISSUE TITLE
Reverse reaction

### ISSUE THREAD

**Issue by DeniseSl22** [PathVisio/pathvisio#52]
Could a double arrow be added to the MIM interactions?
This would portray a reverse reaction (not only visualise a reverse reaction), making the directionality both ways (including the  enzyme connected in both ways).
Now, I have to add two arrows, and let them face in opposite directions, add two anchors, add two enzyme conversion interactions and then overlay everything.
See as an example PW on [pyrimidine metabolism](http://www.wikipathways.org/index.php/Pathway:WP4022).

**Comment by mkutmon** [comment]
@DeniseSl22 well you could change the start line type to Arrow as well. Both sides have a line type. If both are set to Arrow then the interaction is reversible.

**Comment by DeniseSl22** [comment]
Yeah Elisa showed me that, but Ryan was not sure if that meant the same visually and computerwise. 
And I think not a lot of users do not use the back page to change these things, right? They just want a button to click on and then see and use a double arrow ;)

**Comment by mkutmon** [comment]
this is resolved correctly into two reversible reaction in the RDF. So it means the same computational and visual. We could create a template but I wouldn't introduce a new arrow type.

**Comment by ariutta** [comment]
I usually draw these using two interactions. Is it better to do that, or to use just one interaction with two arrowheads?

**Comment by mkutmon** [comment]
I think visually they look the same and semantically they get converted into the same, so from my point of view it doesn't matter how you draw them. Having a reversible conversion template would be nice though.

**Comment by DeniseSl22** [comment]
Okay, I will check this with Ryan.
The template idea sound good as well ;) Perhaps we can include an enzyme and a side metabolite? Like this example (where ADP is the side metabolites, and metabolite 1 and 2 are equilibrium, the enzyme catalyses the reaction, metabolite 2 can be converted to metabolite 1 if ADP is present in this case):

![image](https://user-images.githubusercontent.com/26277832/29652862-be2d5942-88a7-11e7-88d5-8e36f866613e.png)

**Comment by mkutmon** [comment]
well with a side metabolite it gets more tricky - I think then I would draw it as two reactions because otherwise you don't know if ADP is a product of Met1 -> Met2 or Met2 -> Met1. 
I guess it would look something like this:
![image](https://user-images.githubusercontent.com/2158343/29967233-fbb403b4-8f15-11e7-9688-1193464bad97.png)

**Comment by DeniseSl22** [comment]
agreed!

**Comment by DeniseSl22** [comment]
Okay, let's make a simple template for PV4.

**Comment by DeniseSl22** [comment]
Fixed by https://github.com/PathVisio/pathvisio/pull/121/commits/a241447741f8ba8f5d545b48e9c89a3f6067a828
